src/Zend/Crypt/Rsa/Key/Public.php

```diff
@@ -28,15 +28,22 @@
  */
 class Zend_Crypt_Rsa_Key_Public extends Zend_Crypt_Rsa_Key
 {
+    /**
+     * @var string}null
```

Comment:
This should be string|null yeah?

Reply:
Ooo, yeah, surprised that phpstan didn't catch that.